rl_algorithms/common/networks/base_network.py

```diff
@@ -46,11 +47,12 @@ def forward_(self, x: torch.Tensor, n_tau_samples: int = None):
         return x
 
 
-def calculate_fc_input_size(state_dim: tuple, cnn: ConfigDict):
+def calculate_fc_input_size(state_dim: tuple, backbone_cfg: ConfigDict):
```

Comment:
I think this method should move into `BaseNetwork` due to removing if condition.

Reply:
resolved in commit 2cad1a7!!